depends/README.md

```diff
@@ -126,7 +126,6 @@ The following can be set when running make: `make FOO=bar`
 - `LOG`: Use file-based logging for individual packages. During a package build its log file
   resides in the `depends` directory, and the log file is printed out automatically in case
   of build error. After successful build log files are moved along with package archives
-- `LTO`: Use LTO when building packages.
```

Comment:
29185 upstream kept a line in? https://github.com/bitcoin/bitcoin/pull/29185/files#diff-dff7e06f00d3fb1cd0ccf1a491314d52457870824b4cbabfdc4a8ca77ccd7586R126

Reply:
Resolved in latest push!